### PR TITLE
FB Pixel update

### PR DESCRIPTION
**Pull request by ishachopra0401**
## What do these changes do?

Updated FB Pixel doc

### CODE REVIEW

File: docs/destinations/streaming-destinations/fb-pixel.mdx [hunk 1]
```diff
@@ -360,6 +383,34 @@ Facebook Pixel uses ISO 8601 timestamp without the timezone information. It expe
...
+
+The priority order of setting the `content_type` value is as follows:
+
+1. `content_type` provided in the `integrations` object is given the highest priority.
+2. `content_type` provided in the RudderStack dashboard is given the second highest priority.
+3. If none of the above is provided, the `content_type` is set to `product` by default, except in the following cases:
+  - If **Product List Viewed** event is sent:
+      - with `products` array, then the `content_type` is set to `product`.
+      - without the `products` array, then the `content_type` is set to `product_group`.
+  - If **Product Viewed** event is mapped in the RudderStack dashboard, then the `content_type` is set to `product`.
```
Comment (shrouti1507) [inline docs/destinations/streaming-destinations/fb-pixel.mdx:406]: this line is not required as we have already mentioned in line 402

**Review by shrouti1507 — APPROVED** [review]

File: docs/destinations/streaming-destinations/fb-pixel.mdx [hunk 2]
```diff
@@ -360,6 +383,33 @@ Facebook Pixel uses ISO 8601 timestamp without the timezone information. It expe
...
+"integrations": {
+  "FacebookOfflineConversions": {
+    "contentType": "sending dedicated content type for this particular payload"
+  }
+}
+```
+
+RudderStack sets the value of `content_type` in the following priority order:
+
+1. The highest priority is given to the `content_type` provided in the `integrations` object.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/fb-pixel.mdx:400]:
```suggestion
1. RudderStack gives the highest priority to the `content_type` provided in the `integrations` object.
```

File: docs/destinations/streaming-destinations/fb-pixel.mdx [hunk 3]
```diff
@@ -360,6 +383,33 @@ Facebook Pixel uses ISO 8601 timestamp without the timezone information. It expe
...
+
+```javascript
+"integrations": {
+  "FacebookOfflineConversions": {
+    "contentType": "sending dedicated content type for this particular payload"
+  }
+}
+```
+
+RudderStack sets the value of `content_type` in the following priority order:
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/fb-pixel.mdx:398]: Maybe we can remove this sentence, it's quite self-explanatory in the following bullets.

Comment (ishachopra0401) [reply docs/destinations/streaming-destinations/fb-pixel.mdx:398]: Maybe the sentence is not adding much value, but starting bullets without any intro might look weird

**Pull request merged**